### PR TITLE
Melissa's advice page

### PR DESCRIPTION
**Pull request by mprove1**
Added my advice page and updated the index.hml & style.css files.

### CODE REVIEW

**Comment by acodeinprogress** [comment]
Hey @mprove1,

Great work so far! We'd love it if you would add some styles to the style.css file and then link to your html file in the index.html file! Once you do that, you can add, commit, and push and your changes will join your pull request here.

Let me know if you need any help!

**Comment by mprove1** [comment]
I thought I'd done that. Let me try again.



Sent with Mixmax

On Wed, Dec 2, 2015 at 5:13 PM, Sharon < notifications@github.com > wrote:
Hey @mprove1 ,

Great work so far! We'd love it if you would add some styles to the style.css
file and then link to your html file in the index.html file! Once you do that,
you can add, commit, and push and your changes will join your pull request here.

Let me know if you need any help!

—
Reply to this email directly or view it on GitHub .




MELISSA MANOUSOS m : 6 4 6 .3 3 1. 8 4 3 9
e-fax: 8 6 6 .5 2 5. 5 6 7 1
mmanousos@gmail.com
iChat/AIM: mnmproduction

All content is confidential. Do not discuss or disclose except as necessary to
perform your job function. This includes social media.

**Comment by mprove1** [comment]
I think they should be ready to merge now.

**Comment by acodeinprogress** [comment]
Great work, @mprove1! Your code is all merged in.
